pgbouncer/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+***Changed***:
+
+* Upgrade `psycopg2-binary` to `v2.9.8` ([#15949](https://github.com/DataDog/integrations-core/pull/15949))
```

Comment:
This should go under `Added` and this entry should say dropped support for Python 2

Reply:
You still need a Changed entry to document that Agent 6/Python 2 is no longer supported

Reply:
https://datadoghq.dev/integrations-core/guidelines/pr/#changelog-entries

Reply:
Thanks. changelog updated